frontend/apps/marketing/src/contentful/selectors/getExperienceEntryFields.ts

```diff
@@ -0,0 +1,23 @@
+import {Experience} from '@contentful/experiences-sdk-react';
+
+export function getExperienceEntryFieldsFromExperience(
+  experience: Experience | undefined,
+) {
+  return experience?.entityStore?.experienceEntryFields;
+}
+
+export function getSeoMetadataEntryFromExperience(
+  experience: Experience | undefined,
+) {
+  return getExperienceEntryFieldsFromExperience(experience)?.seoMetadata;
+}
+
+export function getSeoMetadataFromExperience(
+  experience: Experience | undefined,
+) {
+  return getSeoMetadataEntryFromExperience(experience)?.fields;
+}
+
+export function getPageHeading(experience: Experience | undefined) {
+  return getExperienceEntryFieldsFromExperience(experience)?.pageHeading;
+}
```

Comment:
May be simplified to this (checked locally, tests still pass successfully):
```suggestion
export function getExperienceEntryFieldsFromExperience(
  experience?: Experience,
) {
  return experience?.entityStore?.experienceEntryFields;
}

export function getSeoMetadataEntryFromExperience(experience?: Experience) {
  return getExperienceEntryFieldsFromExperience(experience)?.seoMetadata;
}

export function getSeoMetadataFromExperience(experience?: Experience) {
  return getSeoMetadataEntryFromExperience(experience)?.fields;
}

export function getPageHeading(experience?: Experience) {
  return getExperienceEntryFieldsFromExperience(experience)?.pageHeading;
}
```

Reply:
When you do:

```
export function getExperienceEntryFieldsFromExperience(
  experience?: Experience,
)
```

It allows for `getExperienceEntryFieldsFromExperience()` which isn't an allowed use case (the explicit union forces you to enter a parameter). There isn't a test for this case as it wasn't an allowed usage.